src/Tribe/Main.php

```diff
@@ -364,38 +376,86 @@ public function maybe_set_common_lib_info() {
+		public function maybe_bail_if_old_et_is_present() {
 			// early check for an older version of Event Tickets to prevent fatal error
-			if (
-				class_exists( 'Tribe__Tickets__Main' ) &&
-				! version_compare( Tribe__Tickets__Main::VERSION, $this->min_et_version, '>=' )
-			) {
-				add_action( 'admin_notices', [ $this, 'compatibility_notice' ] );
-				add_action( 'network_admin_notices', [ $this, 'compatibility_notice' ] );
-				add_filter( 'tribe_ecp_to_run_or_not_to_run', [ $this, 'disable_pro' ] );
-				add_action( 'tribe_plugins_loaded', [ $this, 'remove_exts' ], 0 );
-				/*
-				* After common was loaded by another source (e.g. Event Tickets) let's append this plugin source files
-				* to the ones the Autoloader will search. Since we're appending them the ones registered by the plugin
-				* "owning" common will be searched first.
-				*/
-				add_action( 'tribe_common_loaded', [ $this, 'register_plugin_autoload_paths' ] );
-
-				// Disable older versions of Community Events to prevent fatal Error.
-				remove_action( 'plugins_loaded', 'Tribe_CE_Load', 2 );
+			if ( ! class_exists( 'Tribe__Tickets__Main' ) ) {
+				return;
+			}
 
+			if ( version_compare( Tribe__Tickets__Main::VERSION, $this->min_et_version, '>=' ) ) {
 				return;
 			}
 
-			// WordPress and PHP Version Check
-			if ( ! self::supportedVersion( 'wordpress' ) || ! self::supportedVersion( 'php' ) ) {
-				add_action( 'admin_notices', array( $this, 'notSupportedError' ) );
+			$this->should_prevent_autoload_init = true;
+
+			add_action( 'admin_notices', [ $this, 'compatibility_notice' ] );
+			add_action( 'network_admin_notices', [ $this, 'compatibility_notice' ] );
+			add_filter( 'tribe_ecp_to_run_or_not_to_run', [ $this, 'disable_pro' ] );
+			add_action( 'tribe_plugins_loaded', [ $this, 'remove_exts' ], 0 );
+			/*
+			* After common was loaded by another source (e.g. Event Tickets) let's append this plugin source files
+			* to the ones the Autoloader will search. Since we're appending them the ones registered by the plugin
+			* "owning" common will be searched first.
+			*/
+			add_action( 'tribe_common_loaded', [ $this, 'register_plugin_autoload_paths' ] );
+
+			// if we get in here, we need to reset the global common to ET's version so that we don't cause a fatal
+			$this->reset_common_lib_info_back_to_et();
+
+			// Disable older versions of Community Events to prevent fatal Error.
+			remove_action( 'plugins_loaded', 'Tribe_CE_Load', 2 );
+		}
+
+		/**
+		 * Prevents bootstrapping and autoloading if the version of WP or PHP are too old
+		 *
+		 * @since 4.9.3.2
+		 */
+		public function maybe_bail_if_invalid_wp_or_php() {
+			if ( self::supportedVersion( 'wordpress' ) && self::supportedVersion( 'php' ) ) {
+				return;
+			}
+
+			add_action( 'admin_notices', array( $this, 'notSupportedError' ) );
+
+			// if we get in here, we need to reset the global common to ET's version so that we don't cause a fatal
+			$this->reset_common_lib_info_back_to_et();
```

Comment:
I don't think I can comment on it because it's not something changed in _this_ PR, but I believe this method (`reset_common_lib_info_back_to_et()`, I mean) needs further attention; similar to a previous comment it looks to be populating `$GLOBALS['tribe-common-info']['version']` with PHP source rather than the parsed out version number.

Reply:
This should now be resolved in https://github.com/moderntribe/the-events-calendar/pull/2572/files#diff-e5ad0809a1399fb28c50f3775e36ba7eR394

Reply:
Ah yes, ignore me!